Snippets/SqlTransportNative/SqlTransportNative_1/SqlTransportNative_1.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
I notice here you're multi-targeting a snippets project. What's your rationale for that? We were going back and forth on whether snippets should be allowed (encouraged? enforced?) to multi-target. To ensure that any APIs used are compiled and validated using both targets?

Reply:
The thinking around enforcing no multi-targeting for snippets as well as samples is that there shouldn't be a public API difference between the various assemblies in the package, so there's no reason to take the overhead of compiling multiple versions of the snippets assemblies.

Validating that the API is the same should happen in the project via some sort of API approval, not the snippets.

Reply:
Fair enough. But that conversation should be happening elsewhere. Not on this pr

Can this be merged and when the multi targetting is decided this can be fixed

Reply:
@bording some extra context. there are several snippet projects that use explicit unit tests to generate doco. u might want to confirm the multi target changes dont effect those. i dont think it will, but just to be sure...